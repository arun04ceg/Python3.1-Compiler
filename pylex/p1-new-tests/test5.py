False      class      finally    is         return
