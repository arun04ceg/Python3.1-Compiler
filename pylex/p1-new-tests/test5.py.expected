(KEYWORD False)
(KEYWORD class)
(KEYWORD finally)
(KEYWORD is)
(KEYWORD return)
(NEWLINE)
(ENDMARKER)
